### DISCUSSION TITLE
Search via other search engines in the context menu

### DISCUSSION THREAD

**Discussion by MintMain21** [Floorp-Projects/Floorp#867] [Feature Request]
When I highlight text and open the context menu, I'm given the options to search my default search engine (DuckDuckGo) and search in a private window. I would like the option to search with the other search engines I've collected as well. Otherwise I'd have to use an additional browser extension.

**Comment by Telmesomething** [comment]
I think it'll add too much to the devs daily workload and requests to create something like this will remain at the bottom of the pile especially when there are addons which will complete the task in a matter of seconds. I use one called [ContextSearch web-ext](https://addons.mozilla.org/en-US/firefox/addon/contextsearch-web-ext/) which has the option to pickup the list of the additional search engines you say you've collected already. 

But the task was so much easier before the advent of WebEx and search engines could be added with a click of a button. Now there are several steps necessary to achieve the same goal unfortunately
